modules/covid_vaccine/app/models/covid_vaccine/v0/registration_submission.rb

```diff
@@ -13,6 +13,9 @@ class RegistrationSubmission < ApplicationRecord
 
       attr_encrypted :form_data, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller
       attr_encrypted :raw_form_data, key: Settings.db_encryption_key, marshal: true, marshaler: JsonMarshal::Marshaller
+
+      serialize :form_data, :raw_form_data, :eligibility_info, JSON
```

Comment:
Have you tested the format of the JSON is the same?

Reply:
This has since been changed to use the JsonMarshal::Marshaller class which calls JSON.parse(some_attribute) when retrieving a field and calls .to_json when saving to the database to save as a string format